RecoLocalTracker/SiStripClusterizer/python/SiStripClusterChargeCut_cfi.py

```diff
@@ -1,7 +1,7 @@
 import FWCore.ParameterSet.Config as cms
 
 SiStripClusterChargeCutNone = cms.PSet(
-    value     = cms.double(-1.0)
+    value     = cms.double(0.0)
```

Comment:
Could you add a comment explaining that the value must not be less than zero (and the reason for that)? Thanks.